e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/EhrExtractTest.java

```diff
@@ -38,6 +38,7 @@
     private static final String FROM_ASID = "200000000359";
     private static final String TO_ASID = "918999198738";
     private static final String FROM_ODS_CODE = "GPC001";
+    private static final String FROM_ODS_CODE2 = "GP0001";
```

Comment:
Worth changing the value to make sure those two don't get mixed somehow.

Reply:
Sorry, I'm blind, they are different. Maybe `2` instead of `1`, to make the difference more visible? Not so important, up to you.